clicking on dashboard redirects to dashboard page

Himanshu-Vishwas opened the pull request:
Added a script takeThemTo which redirects to the URL which is passed as an argument.

Changes to be committed:
	modified:   pages/ui_dashboard.html

This task has been done.
I would like add redirect to the other nav-links also.

Himanshu-Vishwas committed:
added a script takeThemTo which redirects to the url which is passed as an arguement.
Changes to be committed:
	modified:   pages/ui_dashboard.html

NtemKenyor commented on pages/ui_dashboard.html:
```diff
@@ -2313,6 +2313,13 @@ <h2>Project Settings</h2>
 	</div>
 </div>
+
+<script>
```
you don't need to create another function called "takeThem" because it already exist. check out line 1735 in "dashboard.js" under vendor/js folder.
You may need to add a  "else if" statement for "dashboard" in dashboard.js

NtemKenyor commented on pages/ui_dashboard.html:
```diff
@@ -1794,7 +1794,7 @@
 			<div class="dropdown-content">
 				<span onclick="takeThemTo('render')"><i class="fa fa-external-link"></i>Render page</span>
-				<span onclick="takeThemTo('ui_dashboard')"><i class="fa fa-external-link"></i>UI Dashboard</span>
+				<span onclick="takeThemTo('dashboard.html')"><i class="fa fa-external-link"></i>Dashboard</span>
```
the parameter we used for the function "takeThemTo" is always a string not a URL. Using 'dashboard' should just be fine.

NtemKenyor reviewed (changes requested):
Great Job. I think you understand exactly what we want but you are not very familiar with some areas/locations of code in this Project.
ui_dashboard.html is linked to dashboard.js and all the javascript code is placed there.

Remove the script tag you added in ui_dashboard.html and change the function parameter to 'dashboard' not 'dashboard.html' so you can easily edit the function takeThemTo(whereTo)" in dashboard.js

Himanshu-Vishwas committed:
added dashboard.js script in head
Changes to be committed:
	modified:   assets/vendor/js/dashboard.js
	modified:   pages/ui_dashboard.html

Himanshu-Vishwas commented:
ok changes have been done. Please re-review the PR.

NtemKenyor reviewed (approved):
Thank you very much for your contribution to this open source project.  We are happy to have you. I would be assigning more issue to you, hope that's ok?

Himanshu-Vishwas commented:
I would love to contribute more.
